code/obj/item/misc_weapons.dm

```diff
@@ -1029,6 +1031,16 @@
 			return ..()
 	..()
 
+/// Checks if the target is facing in some way away from the user. Or they're lying down
+/obj/item/katana/proc/SeverButtStuff(var/mob/living/carbon/human/target, var/mob/user)
+	var/dir_from_target = get_dir(target, user)
+	var/list/valid_dirs = alldirs - target.dir - turn(target.dir, 45) - turn(target.dir, -45)
```

Comment:
just use the code the knife uses?

https://github.com/goonstation/goonstation/blob/ef77f60edcc21430d6f1adc4d888afb3672ab2bd/code/WorkInProgress/actuallyKeelinsStuff.dm#L1246

Reply:
From the code, it looks like it requires both players to be facing the same direction. While that would work (and with a lot less var definitions), it seems like it'd cause some awkwardness depending on the direction. North/south, the player'll have to be right behind them, and (if standing diagonally from someone still counts as a dist of 1) east/west, the player could be in any of the three tiles behind them.

My intent is to make the sword able to cut off someone's butt if the target butt is reasonably accessible by sword, essentially anywhere other than the front.

I can change it if you want, though!